README.md

```diff
@@ -6,11 +6,10 @@ The @salesforce/sf-plugins-core provides utilities for writing [sf](https://gith
 
 ## SfCommand Abstract Class
 
-The SfCommand abstract class extends [@oclif/core's Command class](https://github.com/oclif/core/blob/main/src/command.ts) for examples of how to build a definition.
-) class and adds useful extensions to ease the development of commands for use in the Salesforce Unified CLI.
```

Comment:
It looks like the `for examples of how to build a definition. )` bit was left over from a bad merge some time ago so I'm just removing it because it doesn't make sense.